src/components/button/button.svelte

```diff
@@ -148,6 +148,7 @@
     color: var(--color);
     text-decoration: none;
     padding: var(--leo-button-padding, var(--padding-y) var(--padding-x));
+    min-width: max-content;
```

Comment:
This doesn't mean that buttons won't ever reduce to fit inside their parent does it? Especially if in a flex or block-like full width state?

Reply:
It would. Do we really want the button contents stacking though?

Reply:
I think sometimes we might (if I have two words it might be good to have it reflow to two lines on super small screens).

Out of interest, why do we need this? If it's just for the `fab` button icons, maybe we can add this to the `.fab` class?

Reply:
Went ahead and removed. It was a secondary matter for the PR, so we can pick up in future if needed.